server/liveblog/themes/themes.py

```diff
@@ -70,9 +75,27 @@ def get_local_themes_packages(self):
             for file in glob.glob(embed_folder + '/**/theme.json'):
                 yield json.loads(open(file).read())
 
+    def get_themes(self):
+        res = get_resource_service('themes').get(req=None, lookup={})
+        return res
+
     def update_registered_theme_with_local_files(self):
+        embed_folder = os.path.join(os.path.dirname(os.path.realpath(__file__)),
```

Comment:
You probably already know the _"Don't repeat yourself"_ principle.  Here you can apply the _"Don't repeat myself"_ principle :)
This line already exist a little above